### PR TITLE
document PartitionMappings in partitions concepts page

### PR DESCRIPTION
**Pull request by sryza**

### CODE REVIEW

File: docs/content/concepts/partitions-schedules-sensors/partitions.mdx [hunk 1]
```diff
@@ -226,13 +226,49 @@ height={1618}
...
+                start_offset=-1, end_offset=-1
+            ),
+        )
+    },
+)
+def yesterday_event_stats(events):
+    ...
+```
+
+The available PartitionMappings are included here: <https://docs.dagster.io/_apidocs/partitions#partition-mapping>.
```
Comment (erinkcochran87) [inline docs/content/concepts/partitions-schedules-sensors/partitions.mdx:270]:
```suggestion
Refer to the [API docs](https://docs.dagster.io/_apidocs/partitions#partition-mapping) for a list of available `PartitionMappings`.
```

Stick to using formatted links whenever possible - it reads and looks nicer than just a URL.

File: docs/content/concepts/partitions-schedules-sensors/partitions.mdx [hunk 2]
```diff
@@ -226,13 +226,49 @@ height={1618}
...
+A few rules govern default partition-to-partition dependencies:
 
 - When the upstream asset and downstream asset have the same <PyObject object="PartitionsDefinition" />, each partition in the downstream asset depends on the same partition in the upstream asset.
 - When the upstream asset and downstream asset are both time window-partitioned, each partition in the downstream asset depends on all partitions in the upstream asset that intersect its time window.
 
   For example, if an asset with a <PyObject object="DailyPartitionsDefinition" /> depends on an asset with an <PyObject object="HourlyPartitionsDefinition" />, then partition `2022-04-12` of the daily asset the would depend on 24 partitions of the hourly asset: `2022-04-12-00:00` through `2022-04-12-23:00`.
 
+#### PartitionMappings
+
+You can override these rules by providing a <PyObject object="PartitionMapping" /> when specifying a dependency on an asset. For example, here's how to specify that each partition of a daily-partitioned asset depends on the prior day's partition in an upstream asset:
```
Comment (erinkcochran87) [inline docs/content/concepts/partitions-schedules-sensors/partitions.mdx:238]: > You can override these rules by providing

In this sentence, `these` is an unclear antecedent - it's not immediately clear what `these` refers to. I think that's partially because this sentence starts a new section, so it may take a moment to connect the dots.

Consider something like 

- You can override the default partition dependency rules by... 
- To override the default partition dependency rules, provide a...

I know these suggestions are a bit wordy, but it's clear what it's referring to.

**Comment by sryza** [comment]
Made those changes. Back to you @erinkcochran87!

**Review by erinkcochran87 — APPROVED** [review]
